api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -182,9 +182,7 @@ message SatelliteStreamRequest {
   repeated Framing accepted_framing = 4;
 }
 
-// A request to send commands to a satellite.
-//
-// Next ID: 1
+
```

Comment:
Did you mean to remove the comment? I'm also curious why next ID is 1 and not 3.